spec/models/stats/base_stat_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Stats
+  RSpec.describe BaseStat, type: :model do
+    subject { described_class.new(current_registrations, previous_registrations) }
+
+    # context "#json" do
```

Comment:
Again, just a `# TODO: ` comment here to remind us that this test needs to be completed at some point would be useful.

Reply:
Comment added as requested.